plugin-process-to-cgroup-bridge/src/tests.rs

```diff
@@ -0,0 +1,437 @@
+#[cfg(test)]
+mod tests {
```

Comment:
You don't need the `mod tests` here, it's already a module :)
In Rust, each file is a module.